src/usr/local/www/system_authservers.php

```diff
@@ -103,11 +112,7 @@
 	$config['system']['authserver'] = array();
 }
 
-$a_servers = auth_get_authserver_list();
-
-foreach ($a_servers as $servers) {
-	$a_server[] = $servers;
-}
+$a_server = gen_auth_server_array();
```

Comment:
Looking at that original code, I'm not sure why it was done that way. That code, and the new function, could be replaced by `$a_server = array_values(auth_get_authserver_list());` and it would be a lot simpler.